src/uct/api/uct.h

```diff
@@ -1088,6 +1099,12 @@ struct uct_ep_params {
      * Callback that will be invoked when the endpoint is disconnected.
      */
     uct_ep_disconnect_cb_t              disconnect_cb;
+
+    /**
+     * Index of the path which the endpoint should use, must be in the range
+     * 0..@ref uct_iface_attr_t.dev_num_paths - 1.
```

Comment:
do we want to make it unsigned ? So potentially in future we can pass and option to UCT to pick own path ?

Reply:
still insist on unsigned ? IMHO, almost always bad idea.

Reply:
we can let UCT pick own path in the future, by not setting UCT_EP_PARAM_FIELD_PATH_INDEX flag in field_mask